### DISCUSSION TITLE
Switch to Github Actions from Travis and Appveyor

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/CoinUtils#159] [General]
This is to announce some changes that are being made to the CI infrastructure. Due to the closure of Bintray and to improve our CI infrastructure overall, we are moving to Github Actions from Travis and to the use of Github Releases for distributing binaries. Please let us know if you have any feedback!

Update: The switch is now complete!

**Comment by svigerske** [comment]
My feedback is that I'm happy to "see" @xmunoz again! Hello!

**Comment by tkralphs** [comment]
We now have a tentative workflow for attaching binaries to tags. If anyone would like to test and give feedback, that would be fantastic. Please see 

https://github.com/coin-or/CoinUtils/releases/tag/test-tag
